packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -345,8 +366,13 @@ export default class Calendar extends React.Component {
     if (!this.props.forceShowMonthNavigation && allDaysDisabledBefore(this.state.date, 'month', this.props)) {
       return
     }
-    return <a
-      className={cx(['react-datepicker-navigation', 'react-datepicker-navigation--previous'])}
+
+    return <Button
+      className={cx('react-datepicker-navigation--previous')}
+      icon={<span className={cx('prev-month-icon')} />}
+      isIconOnly
+      variant="utility"
+      text="Previous Month"
```

Comment:
Todo: Get translation

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/866/commits/6ea023ec6abb090fc1b5d30e23041f9ddbddad13